pkg/spire/spire_test.go

```diff
@@ -388,46 +396,42 @@ func TestSpire_TaskRunResultsSign(t *testing.T) {
 	}
 
 	for _, tt := range testCases {
-		ctx := context.Background()
-		for _, tr := range tt.taskRunList {
-			if !tt.skipEntryCreate {
-				if !tt.wrongPodIdentity {
-					resp := &fakeworkloadapi.X509SVIDResponse{
-						Bundle: ca.X509Bundle(),
-						SVIDs:  makeX509SVIDs(ca, spiffeid.RequireFromPath(td, getTaskrunPath(tr))),
-					}
-					wl.SetX509SVIDResponse(resp)
-				} else {
-					resp := &fakeworkloadapi.X509SVIDResponse{
-						Bundle: ca.X509Bundle(),
-						SVIDs:  makeX509SVIDs(ca, fooID),
+		t.Run(tt.desc, func(t *testing.T) {
+			ctx := context.Background()
+			for _, tr := range tt.taskRunList {
+				if !tt.skipEntryCreate {
+					if !tt.wrongPodIdentity {
+						resp := &fakeworkloadapi.X509SVIDResponse{
+							Bundle: ca.X509Bundle(),
+							SVIDs:  makeX509SVIDs(ca, spiffeid.RequireFromPath(td, getTaskrunPath(tr))),
+						}
+						wl.SetX509SVIDResponse(resp)
+					} else {
+						resp := &fakeworkloadapi.X509SVIDResponse{
+							Bundle: ca.X509Bundle(),
+							SVIDs:  makeX509SVIDs(ca, fooID),
+						}
+						wl.SetX509SVIDResponse(resp)
 					}
-					wl.SetX509SVIDResponse(resp)
 				}
```

Comment:
While you're at it (because no good deed goes unpunished!) you might fix the violations of https://google.github.io/styleguide/go/decisions#data-driven-test-cases here. It's probably easier to make a helper function that returns a `*fakeworkloadapi.X509SVIDResponse` and save that in the test case struct instead of the existing `SVIDResponse`-related fields.

Reply:
Agreed... I'm going to do this in a test cleanup PR separate from the lint fixes.